mkt/search/tests/test_filters.py

```diff
@@ -52,14 +52,14 @@ def test_q(self):
             in should)
         ok_({'prefix': {'name': {'boost': 1.5, 'value': 'search terms'}}}
             in should)
-        ok_({'match': {'name_english': {'query': 'search terms',
-                                        'boost': 2.5}}}
-            in should)
-        ok_({'match': {'description_english': {'query': 'search terms',
-                                               'boost': 0.6,
-                                               'analyzer': 'english_analyzer',
-                                               'type': 'phrase'}}}
+        ok_({'match': {'name_l_english': {'query': 'search terms',
+                                          'boost': 2.5}}}
             in should)
+        ok_({'match': {'description_l_english':
+            {'query': 'search terms',
+             'boost': 0.6,
+             'analyzer': 'english_analyzer',
+             'type': 'phrase'}}} in should)
```

Comment:
I find the nesting on this confusing but I suppose it's fine if that's the best it can get.